rules/windows/sysmon/sysmon_psexec_pipes_artifacts.yml

```diff
@@ -0,0 +1,29 @@
+title: PsExec Pipes Artifacts
+id: 9e77ed63-2ecf-4c7b-b09d-640834882028
+status: experimental
+description: Detecting use PsExec via Pipe Creation/Access to pipes
+author: Nikita Nazarov
```

Comment:
Hello @NikitaStormwind !

Thank you for your contribution.
Would you mind adding "oscd.community" to the author field?
It will help with the promotion of the initiative and engage more people in the next sprints.

Reply:
Done